io.itch.doctorm64.skippybot.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: io.itch.doctorm64.skippybot
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --share=ipc
+
+# sound and graphics
+- --socket=pulseaudio
+- --device=dri
+
+command: skippybot
+modules:
+  - name: skippybot
+    build-options:
+      env:
+        SKIPPY_VERSION: "115"
+    buildsystem: simple
+    build-commands:
+      # Game
+      - install -Dm644 UnityPlayer.so /app/bin/UnityPlayer.so
+      - install -Dm755 skippy_${SKIPPY_VERSION}.x86_64 /app/bin/skippybot
+      - cp -r skippy_${SKIPPY_VERSION}_Data/ /app/bin/skippybot_Data
+      # meta files
+      - install -Dm644 $FLATPAK_ID.appdata.xml /app/share/metainfo/$FLATPAK_ID.appdata.xml
+      - install -Dm644 $FLATPAK_ID.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+    sources:
+      - type: archive
+        # redirected URL because itch.io does not allow direct downloads of assets. We are checking the hash tho, so file is the same.
+        url: https://cdn.matix-media.net/dd/595efc05
+        dest-filename: skippy_linux64_$SKIPPY_VERSION.zip
```

Comment:
This make the file saved as `skippy_linux64_$SKIPPY_VERSION.zip`.

I don't even see the reason to do so.

Reply:
Ah, thanks for the notice, didn't catch that.
Changed it locally to just use the actual version number instead.